### ISSUE TITLE
parseErrorStack is not compatible with React Native 0.64

### ISSUE THREAD

**Issue by leethree** [Instabug/Instabug-React-Native#594]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

I'm getting a lot of errors from Instabug when reporting errors using `CrashReporting.reportJSException(err)`. 

```
TypeError: n.split is not a function. (In 'n.split('\n')', 'n.split' is undefined)
```

After investigation, it turns out React Native 0.64 changed the internal method `parseErrorStack` to accepts a `string` instead of a `Error`. See this commit here: 

https://github.com/facebook/react-native/commit/9edfc43aad2f68be8a37c916779681cd9ad13fee#diff-6c06cffe52fa76adfc945f6d386e1635701525b3299c0f3ce1bdf80a479e3922

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/instabug-reactnative/utils/InstabugUtils.js b/node_modules/instabug-reactnative/utils/InstabugUtils.js
index 99c22f9..0ad0b9a 100644
--- a/node_modules/instabug-reactnative/utils/InstabugUtils.js
+++ b/node_modules/instabug-reactnative/utils/InstabugUtils.js
@@ -6,7 +6,7 @@ import InstabugConstants from './InstabugConstants';
 import parseErrorStackLib from '../../react-native/Libraries/Core/Devtools/parseErrorStack.js';
 
 export const parseErrorStack = (error) => {
-  return parseErrorStackLib(error);
+  return parseErrorStackLib(error?.stack);
 };
 
 const originalHandler = global.ErrorUtils.getGlobalHandler();
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by AliAbdelfattah** [comment]
Thank you for reporting this. We're looking into it.

**Comment by SudoPlz** [comment]
Also, for those of us that are not using your crash reports, why do you force that code to run without our permission?
Why not ONLY parse the js errors if we're using your crash reporting?

Thank you

**Comment by AliAbdelfattah** [comment]
@SudoPlz makes sense. We'll look into that as well, thank you for mentioning it.

**Comment by SudoPlz** [comment]
While waiting for the fix we actually deactivated instabug js crash reporting by patching the instabug module ourselves:

**instabug-reactnative+10.4.0.patch:**

	diff --git a/node_modules/instabug-reactnative/utils/InstabugUtils.js b/node_modules/instabug-reactnative/utils/InstabugUtils.js
	index 99c22f9..72c8a23 100644
	--- a/node_modules/instabug-reactnative/utils/InstabugUtils.js
	+++ b/node_modules/instabug-reactnative/utils/InstabugUtils.js
	@@ -9,7 +9,6 @@ export const parseErrorStack = (error) => {
	   return parseErrorStackLib(error);
	 };
	 
	-const originalHandler = global.ErrorUtils.getGlobalHandler();
	 var _isOnReportHandlerSet = false;
	 
	 
	@@ -45,44 +44,7 @@ export const isOnReportHandlerSet = () => {
	 };
	 
	 export const captureJsErrors = () => {
	-  if (!process.env.JEST_WORKER_ID) {
	-    if (__DEV__) {
	-      return;
	-    }
	-  }
	-
	-  function errorHandler(e, isFatal) {
	-    let jsStackTrace = parseErrorStackLib(e);
	-
	-    //JSON object to be sent to the native SDK
	-    var jsonObject = {
	-      message: e.name + " - " + e.message,
	-      os: Platform.OS,
	-      platform: 'react_native',
	-      exception: jsStackTrace
	-    }
	-
	-    if (Platform.OS === 'android') {
	-      if (_isOnReportHandlerSet) {
	-        IBGEventEmitter.emit(InstabugConstants.SEND_UNHANDLED_CRASH, jsonObject);
	-      } else {
	-        Instabug.sendJSCrash(JSON.stringify(jsonObject));
	-      }
	-    } else {
	-      Instabug.sendJSCrash(jsonObject);
	-    }
	-
	-    if (originalHandler && !process.env.JEST_WORKER_ID) {
	-      if (Platform.OS === 'ios') {
	-        originalHandler(e, isFatal);
	-      } else {
	-        setTimeout(() => {
	-          originalHandler(e, isFatal);
	-        }, 500);
	-      }
	-    }
	-  }
	-  global.ErrorUtils.setGlobalHandler(errorHandler);
	+  return ;
	 };
	 
	 export default {

**Comment by AliAbdelfattah** [comment]
Fix released in v10.8.1.
